core/indexer/fulltext.go

```diff
@@ -36,6 +39,66 @@ func (i *indexer) ForceFTIndex() {
 	}
 }
 
+func (i *indexer) getSpaceIdsByPriority() []string {
+	var ids = make([]string, 0, i.lastSpacesSubscription.Len())
+	i.lastSpacesSubscription.Iterate(func(_ string, v *types.Struct) bool {
+		id := pbtypes.GetString(v, bundle.RelationKeyTargetSpaceId.String())
+		if id != "" {
+			ids = append(ids, id)
+		}
+		return true
+	})
+
+	log.Warnf("ft space ids priority: %v", ids)
+	return ids
+}
+
+func (i *indexer) updateSpacesPriority(priority []string) {
+	techSpaceId := i.techSpaceId.TechSpaceId()
+
+	priority = append([]string{techSpaceId}, slices.DeleteFunc(priority, func(s string) bool {
+		return s == techSpaceId
+	})...)
+	log.Warnf("update spaces priority: %v", priority)
+
+	i.spaceReindexQueue.UpdatePriority(priority)
+}
+
+func (i *indexer) subscribeToSpaces() error {
+	objectReq := subscription.SubscribeRequest{
+		SubId:             fmt.Sprintf("lastOpenedSpaces"),
```

Comment:
why?

Reply:
because we want to prioritize last opened spaces to be indexed first. This is the whole idea

Reply:
ah, got you now

Reply:
fixed